app/views/books/show.html.erb

```diff
@@ -1,10 +1,26 @@
 <div>
   <h1>Title</h1>
-  <%= book.title %>
-  <h1>ISBN</h1>
+  <h1><%= book.title %></h1>
+  <h3>ISBN</h3>
   <%= book.isbn %>
-  <h1>Year of Published</h1>
+  <h3>Year of Published</h3>
   <%= book.year %>
-  <h1>Author</h1>
+  <h3>Author</h3>
   <%= "#{book.author.first_name}  #{book.author.last_name}"%>
+
+  <h1>Comment List</h1>
+  <% book.comments.each do |comment| %>
+    <%= "#{comment.user.email}: '#{comment.text}' - #{comment.rating} stars" %>
+  <% end %>
+
+  <h1>Add Comment</h1>
+  <%= form_for([book, @new_comment]) do |c| %>
```

Comment:
Just some info here:
- You could do `<%= form_for([book, book.comments.new]) do |c| %>` and don't create an instance variable on the controller.
- Check `remote: true` here: http://guides.rubyonrails.org/working_with_javascript_in_rails.html#form-for. This will generate a JS request where you could add the comment if you want without reloading the screen.

This are just tips so you know 😄